core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client}
+  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client.  This protocol falls back to Java Web Start Agent Protocol/2 (insecure) when it can't create a secure connection.  This protocol is not recommended.  Use Java Web Start Agent Protocol/4 instead.}
```

Comment:
I might need help with that, I'm not sure I completely understand.  I can give you write access to my repository if you want to collaborate.